docs/reference/dqm.md

```diff
@@ -0,0 +1,79 @@
+# Data Quality Monitoring
+
+Data Quality Monitoring (DQM) is a Feast module aimed to help users to validate their data with the user-curated set of rules.
+Validation could be applied during:
+* Historical retrieval (training dataset generation)
+* [planned] Writing features into an online store
+* [planned] Reading features from an online store
+
+Its goal is to address several complex data problems, namely:
+* Data Consistency - new training dataset could be significantly different from the previous, which might require a change in model architecture.
+* Issues/bugs in the upstream pipeline - bug in upstream could cause invalid values to overwrite existing valid values in an online store.
+* Training/serving skew - distribution shift could significantly decrease the performance of the model.
+
+> By “monitoring data quality” we understand verifying that the characteristics of the tested dataset (we call this characteristics dataset's profile) are "equivalent" to the characteristics of the reference dataset.
+> Eg, data currently passed to the model hasn’t changed significantly since the model was trained, and expectations implicitly made by ML algorithm during training are still met.
+> How exactly profiles equivalency should be measured is up to the user. 
+
+### Overview
+
+The validation process consists of the next steps:
+1. User prepares reference dataset (currently only [saved dataset](../getting-started/concepts/dataset.md) from historical retrieval is supported).
+2. User defines profiler function, which should produce profile by given dataset (currently only profilers based on [Great Expectations](https://docs.greatexpectations.io) are allowed).
+3. Validation of tested dataset is performed with reference dataset and profiler provided as parameters.
+
+### Preparations
+Feast with DQM support can be installed via
+```shell
+pip install 'feast[dqm]'
+```
+
+### Dataset profile
+Currently, Feast supports only [great expectation's](https://greatexpectations.io/) [ExpectationSuite](https://legacy.docs.greatexpectations.io/en/latest/autoapi/great_expectations/core/expectation_suite/index.html#great_expectations.core.expectation_suite.ExpectationSuite)
```

Comment:
Is this normal that we already make reference to the "legacy doc" of GE ? Referring to `https://legacy.docs.greatexpectations.io`

Reply:
Link `API Reference` from `docs.greatexpectations.io` leads to this legacy documentation. see https://docs.greatexpectations.io/docs/reference/api_reference/